custom/icds_reports/ucr/data_sources/awc_mgt_forms.json

```diff
@@ -87,11 +87,13 @@
         "transform": {},
         "is_nullable": true,
         "expression": {
-          "location_id_expression": {
+          "location_id": {
             "type": "named",
             "name": "user_location_id"
           },
-          "type": "location_parent_id"
+          "location_type": "block",
+          "location_propery": "_id",
+          "type": "ancestor_location"
```

Comment:
I think this is only useful if you also change the expression for `state_id` otherwise the caching isn't re-used.

Reply:
Sure